README.md

```diff
@@ -12,29 +10,47 @@ https://iabtechlab.com/openrtb
 
-Release branches are created for each monthly release and the history of these can be reviewed on GitHub. The master branch for the repository will always reflect the most recent release, whereas ongoing development work occurs in the 'develop' branch.
+The format for version numbering includes major and minor version and a date code. For example, 2.6-202211 represents the release for November 2022. The following releases may be 2.6-202212 (December), 2.6-202301 (January), etc.
 
 This versioning policy is a break from historical practice for OpenRTB 2.x. In versions of OpenRTB prior to 2.6, major version numbers represent breaking changes and minor version numbers represent non-breaking changes.
 
+#### How To Contribute
+
+1. Clone this GitHub repo into your own GitHub account
+1. Create a new branch based on `develop`, giving your new branch a short but descriptive name (e.g. if you're adding support for a new flux capacitor object, you could call the branch "add-flux-capacitor")
+1. Make the desired changes in your branch, with one commit per logical change (e.g. if you're adding 2 distinct features in your branch, create 2 distinct commits). Give each of your commits a short but descriptive "Summary" name, and then provide a longer "Description" to fully explain your proposed changes.
+1. (Optional) Consider doing a round of internal reviews/feedback within your own organization, and make any additional updates in your own branch.
+1. Once you're happy with your branch, publish it to GitHub. Then create a new Pull Request (PR) to propose merging your changes into the `develop` branch.
+1. The Programmatic Supply Chain Working Group and Commit Group will review your update(s), leave comments, and may propose changes. You may need to make additional commits to receive approval for your PR.
+1. Once your PR is approved, it will be merged into the `main` branch at the time of the next monthly release. Details below on how the Release Process works.
+1. (Optional) If your PR has been open for a long time, it's possible that it cannot be automatically merged into the `develop` branch. In this case, there will be a message in the PR asking you to resolve conflicts before it can be merged.
+
+#### Monthly Release-Cutting Process (for repo admins)
+
+Over the course of each month, the Programmatic Supply Chain Working Group and Commit Group may review any submitted PRs and take the following possible actions for each:
+- approve it for inclusion in the next release
+- ask the author(s) for additional changes
+- reject it (with a rationale)
+
+During the last week of the month, if there are any approved PRs in the `develop` branch, the following steps are executed:
```

Comment:
Nitpick: what happens if the month ends on a Monday? Suggest "last _full_ week"?